src/components/LightUpChinatown/DonationProgressBar.tsx

```diff
@@ -61,44 +63,39 @@ const Filler = styled.div`
   background: #f6c342;
 `;
 
-const GoalImage1 = styled.div`
+const GoalImage = styled.div`
   width: 25px;
   height: 25px;
   background: #ffffff;
   border: 4px solid #ff3a2f;
   box-sizing: border-box;
   position: absolute;
-  left: 50%;
   bottom: 0;
   border-radius: 12px;
 `;
-const GoalImage2 = styled(GoalImage1)`
-  left: 98%;
-  @media (max-width: 920px) {
-    left: 92%;
-  }
+
+const GoalImage1 = styled(GoalImage)`
+  left: calc(50% - 25px);
```

Comment:
What's this offset represent?

Reply:
It doesn't line up with the width of the % yellow progress bar because of the 25px width of the circle image

Reply:
If this were done in scss, I'd say to abstract this into a variable called `$progress-bar-circle-width = 25px` and use it in both places